actionview/lib/action_view/helpers/tags/base.rb

```diff
@@ -120,7 +120,7 @@ def sanitized_value(value)
         def select_content_tag(option_tags, options, html_options)
           html_options = html_options.stringify_keys
           add_default_name_and_id(html_options)
-          options[:include_blank] ||= true unless options[:prompt] || select_not_required?(html_options)
+          options[:include_blank] ||= true unless options.key?(:include_blank) || options[:prompt] || select_not_required?(html_options)
```

Comment:
I have a hard time understanding complex conditions with `unless` but that could be just me.

//cc @carlosantoniodasilva @rafaelfranca @sgrif